catroid/src/main/AndroidManifest.xml

```diff
@@ -243,6 +243,11 @@
         <service android:name=".transfers.MediaDownloadService" />
         <service android:name="com.parrot.freeflight.service.DroneControlService" />
         <service android:name=".utils.StatusBarNotificationManager$NotificationActionService" />
-
-    </application>
+        <service
+            android:name=".ui.cloudmessaging.CustomFirebaseMessagingService">
+            <intent-filter>
+                <action android:name="com.google.firebase.MESSAGING_EVENT"/>
+            </intent-filter>
+        </service>
```

Comment:
Is it possible to somehow deactivate firebase-messaging entirely?
So that the service isn't register in the standalone version.